src/multiselectMixin.js

```diff
@@ -709,6 +709,24 @@ export default {
         this.preferredOpenDirection = 'above'
         this.optimizedHeight = Math.min(spaceAbove - 40, this.maxHeight)
       }
+    },
+    /**
+     * Clears the multiselect value/s
+     */
+    clearValue () {
+      /* istanbul ignore else */
+      if (this.disabled) return
+      /* istanbul ignore else */
+      if (!this.allowEmpty && this.internalValue.length <= 1) {
+        this.deactivate();
+        return
+      }
+      console.log('clear multiselect', this.internalValue);
+      for(let option of this.internalValue) {
+        this.$emit('remove', option, this.id);
+      }
+      /* istanbul ignore else */
+      if (this.closeOnSelect && shouldClose) this.deactivate()
```

Comment:
`shouldClose` is not defined

Reply:
Thanks, I saw this too, @shentao do we need `shouldClose` in here?